fastly/kinesis_test.go

```diff
@@ -0,0 +1,280 @@
+package fastly
+
+import "testing"
+
+func TestClient_Kineses(t *testing.T) {
+	t.Parallel()
+
+	var err error
+	var v *Version
+	record(t, "kineses/version", func(c *Client) {
+		v = testVersion(t, c)
+	})
+
+	// Create
+	var kinesis *Kinesis
+	record(t, "kineses/create", func(c *Client) {
+		kinesis, err = c.CreateKinesis(&CreateKinesisInput{
+			ServiceID:         testServiceID,
+			ServiceVersion:    v.Number,
+			Name:              "test-kinesis",
+			StreamName:        "stream-name",
+			Region:            "us-east-1",
+			AccessKey:         "AKIAIOSFODNN7EXAMPLE",
+			SecretKey:         "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+			Format:            "format",
+			FormatVersion:     2,
+			ResponseCondition: "",
```

Comment:
As per my comment earlier, I don't see `response_condition` as part of the Kinesis stream logging API reference, so I presume this should be removed. Unless the api documentation itself is outdated, but then your recorded fixture file also doesn't include it, so I feel confident this is not needed.

Reply:
To avoid repetition let's get this ResponseCondition/response_condition field deleted everywhere in the PR.